skipper-go/contracts/src/Multihop.sol

```diff
@@ -115,6 +122,8 @@ contract Multihop is Ownable {
      */
     function withdrawNativeBalance() external onlyOwner {
         uint256 balance = address(this).balance;
-        payable(owner()).transfer(balance);
+
+        (bool success, ) = owner().call{value: balance}("");
+        if (!success) revert WithdrawNativeBalanceFailed();
```

Comment:
`.transfer` has a hardcoded 2300 gas limit which prevents smart contracts from doing things like sending ether to a gnosis safe due to out of gas. `.call` instead forwards all gas and is now the recommended way to send ether from a contract. It's also reentrancy safe here since the function is only accessible by the owner.